SPECS/tdnf/tdnf.spec

```diff
@@ -18,21 +18,21 @@ Source2:        cache-updateinfo.service
 Source3:        cache-updateinfo.timer
 Source4:        tdnfrepogpgcheck.conf
 
-Patch0:         tdnf-fix-distroverpkg-search.patch
-Patch1:         tdnf-ssl-support.patch
-Patch2:         tdnf-add-download-command.patch
-Patch3:         tdnf-add-showorder-argument.patch
-Patch4:         tdnf-add-mariner-release.patch
-Patch5:         tdnf-support-multiple-gpgkeys.patch
-Patch6:         tdnf-add-download-no-deps-command.patch
-Patch7:         tdnf-use-custom-keyring-for-gpg-checks.patch
+
 Patch8:         tdnf-mandatory-space-list-output.patch
 
+#Cmake requires binutils
+BuildRequires:  binutils
```

Comment:
These are undeclared requirements that are already included in the worker chroot. However, when trying to build by itself (with no worker chroot), these packages would be missing.